### PR TITLE
2nd Scenario

### PR DESCRIPTION
**Pull request by jGleitz**
I wrote a second scenario. As I worked on this text for more than an hour, it’s likely I miss some mistakes in it. So I’d appreciate sophisticated proof reading.

@m1cm1c: Could you check whether #218 is obsolete after this PR? If not, I’ll happly import your changes once they made it into master.

### CODE REVIEW

**Comment by acBerger** [comment]
[PDF] (http://jgleitz.github.io/Beagle/branches/2nd-screnario/Requirements%20Specification.pdf)

**Comment by RomanLangrehr** [comment]
First of all: :+1: for another scenario :)

File: doc/Requirements Specification/Parts/Models.lyx [hunk 1]
```diff
@@ -426,13 +809,31 @@ status open
 
 
 \backslash
-gls{PCM}
+glslabel{system}{software system}
```
Comment (RomanLangrehr) [inline doc/Requirements Specification/Parts/Models.lyx]: Something went wrong here. The PDF:

>  to assure the Beaglesystemsoftware system will

Comment (acBerger) [inline doc/Requirements Specification/Parts/Models.lyx]: > Beaglesystemsoftware system

This sounds quite funny, but I don't think this belongs like this ;)

File: doc/Requirements Specification/Parts/Models.lyx [hunk 2]
```diff
@@ -426,13 +809,31 @@ status open
...
 \end_layout
 
 \end_inset
 
 .
- The little changes in the 
+ When analysing the imported model, they recognise significant differences
+ between the initial predictions for the prototype and the ones made for
+ the actual implementation.
+ The necessity to check copy right violations and content integrity as well
```
Comment (RomanLangrehr) [inline doc/Requirements Specification/Parts/Models.lyx]: [copyright](http://dict.leo.org/ende/index_de.html#/search=copyright&searchLoc=0&resultOrder=basic&multiwordShowSingle=on) is one word.

File: doc/Requirements Specification/Parts/Models.lyx [hunk 3]
```diff
@@ -116,46 +116,37 @@ The chapter should contain:
...
+\begin_inset Foot
+status collapsed
 
 \begin_layout Plain Layout
-all the people, living life in peace! (You-hu) You may say I'm a dreamer,
- but I'm not the only one.
- I hope one day you'll join us and the world will be as one.
+All characters and organisations appearing in this work are fictitious.
+ Any resemblance to real persons or companies, living or dead, is purely
+ coincidental
```
Comment (RomanLangrehr) [inline doc/Requirements Specification/Parts/Models.lyx]: Missing dot in the end.

In the PDF, the footnote has the number "2". Why?!

**Comment by acBerger** [comment]
lgtm

File: doc/Requirements Specification/Parts/Models.lyx (hunk 1 above)
Comment (jGleitz) [inline doc/Requirements Specification/Parts/Models.lyx]: Oh no. I thought I had managed to fix this! Can somebody explain why this is happening?

File: doc/Requirements Specification/Parts/Models.lyx (hunk 3 above)
Comment (jGleitz) [inline doc/Requirements Specification/Parts/Models.lyx]: Because footnote 1 is on page 29, footnote 2 is on page 31

**Comment by jGleitz** [comment]
Fixed all of the mentioned issues.

File: doc/Requirements Specification/Parts/Models.lyx (hunk 3 above)
Comment (RomanLangrehr) [inline doc/Requirements Specification/Parts/Models.lyx]: Ah, okay...

**Comment by jGleitz** [comment]
fixes #88

**Comment by RomanLangrehr** [comment]
Lgtm

**Comment by acBerger** [comment]
lgtm

**Pull request merged**